lib/matplotlib/_constrained_layout.py

```diff
@@ -188,7 +283,7 @@ def _get_margin_from_padding(obj, *, w_pad=0, h_pad=0,
     return margin
 
 
-def _make_layout_margins(fig, renderer, *, w_pad=0, h_pad=0,
+def _make_layout_margins(_layoutgrids, fig, renderer, *, w_pad=0, h_pad=0,
```

Comment:
Changes from here and below are just adding the `_layoutgrids` structure to all the methods....